nodejs/week1/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "week1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node avg.js"
```

Comment:
As you did not need any `npm` dependencies for your code, there is no need to make this code into an `npm` package through `npm init`. It would have been enough to just run your code via `node avg.js` in the terminal.